external-applications/quarkus-workshop-super-heroes/src/test/resources/villain.yaml

```diff
@@ -23,12 +23,18 @@ items:
         git:
           uri: https://github.com/quarkusio/quarkus-workshops.git
         type: Git
+        configMaps:
+        - configMap:
+            name: settings-mvn
+          destinationDir: "configuration"
       strategy:
         type: Source
         sourceStrategy:
           env:
             - name: ARTIFACT_COPY_ARGS
               value: -p -r lib/ *-runner.jar
+            - name: MAVEN_ARGS
+              value: -s /opt/app-root/src/configuration/settings.xml -Dquarkus.version=${version.plugin.quarkus} -DskipTests=true
```

Comment:
Adding `-s /opt/app-root/src/configuration/settings.xml` is a workaround as the instructions I've found for overwriting the `settings.xml` in OCP didn't work. 

Also I have to skip the tests as they are using docker.